packages/0x.js/webpack.config.js

```diff
@@ -49,6 +49,8 @@ module.exports = {
                         loader: 'awesome-typescript-loader',
                         query: {
                             declaration: false,
+                            declarationMap: false,
```

Comment:
These new `query` fields are overrides for the new default fields for the `tsconfig.json` at the root of the monorepo. We need to override these fields because they are incompatible with awesome-typescript-loader/webpack.

Reply:
Can you add this comment to the file itself? Feel like other devs will wonder about these in the future.